AlphaWallet/Accounts/Views/AccountViewTableSectionHeader.swift

```diff
@@ -12,28 +12,26 @@ class AccountViewTableSectionHeader: UIView {
         var title: String {
             switch self {
             case .hdWallet:
-                return R.string.localizable.walletTypesHdWallets()
+                return R.string.localizable.walletTypesHdWallets().uppercased()
             case .keystoreWallet:
-                return R.string.localizable.walletTypesKeystoreWallets()
+                return R.string.localizable.walletTypesKeystoreWallets().uppercased()
             case .watchedWallet:
-                return R.string.localizable.walletTypesWatchedWallets()
+                return R.string.localizable.walletTypesWatchedWallets().uppercased()
```

Comment:
The title text has to be changed too

Reply:
@hboon you mean that "HD Wallets" should be changed to "YOUR WALLETS"?
do i need to remove .uppercased() and update just localization file?

Reply:
Keep .uppercased() (but not terribly important which way) and update localisation file to "Your Wallets". Remember to check if the strings are used elsewhere.

Reply:
They are using only in HeaderType enumeration there i no information now "Keystore Wallets" should be changed, do i need to change it?

Reply:
Yes, that too.

Reply:
What new name for Keystore Wallets should be? maybe i missed something and cant find.

Reply:
Oh. Just keep it as "Keystore Wallets" (.uppercase()). Sorry.